src/App.js

```diff
@@ -24,10 +24,11 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
-                const selectedTags = insights.chrome?.mapGlobalFilter?.(data)?.filter(item => !item.includes('Workloads')) || undefined;
+                const [workloads, SID, selectedTags] = insights.chrome?.mapGlobalFilter?.(data, false, true);
```

Comment:
I think this will crash if `insights.chrome?.mapGlobalFilter?.(data, false, true);` returns undefined. You should add `[]` fallback.